cli/src/commands/merkle-tree/initialize.ts

```diff
@@ -36,9 +41,15 @@ class InitializeCommand extends Command {
 
   async run() {
     const { flags } = await this.parse(InitializeCommand);
-    const { pubKey, message, transaction } = flags;
+    const {
+      oldMerkleTreePubkeyString,
+      newMerkleTreePubkeyString,
+      message,
+      transaction,
+    } = flags;
 
-    const merkleTreeKey = new PublicKey(pubKey);
+    const oldMerkleTreePubkey = new PublicKey(oldMerkleTreePubkeyString);
+    const newMerkleTreePubkey = new PublicKey(newMerkleTreePubkeyString);
```

Comment:
you can parse the flag as a public key then you don't need to convert manually down here:
```
    newMerkleTreePubkey: Flags.string({
      char: "n",
      parse: (nMt) => new PublicKey(nMt),
```

Reply:
Seems like I can't , actually

```
src/commands/merkle-tree/initialize.ts:18:7 - error TS2769: No overload matches this call.
  The last overload gave the following error.
    Type '(k: String) => PublicKey' is not assignable to type 'FlagParser<string, string, CustomOptions>'.
      Type 'PublicKey' is missing the following properties from type 'Promise<string>': then, catch, finally

18       parse: (k: String) => new PublicKey(k),
         ~~~~~

  node_modules/@oclif/core/lib/interfaces/parser.d.ts:322:5
    322     (options?: P & Partial<OptionFlag<T>>): OptionFlag<T | undefined>;
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    The last overload is declared here.

src/commands/merkle-tree/initialize.ts:24:7 - error TS2769: No overload matches this call.
  The last overload gave the following error.
    Type '(k: String) => PublicKey' is not assignable to type 'FlagParser<string, string, CustomOptions>'.
      Type 'PublicKey' is not assignable to type 'Promise<string>'.

24       parse: (k: String) => new PublicKey(k),
         ~~~~~

  node_modules/@oclif/core/lib/interfaces/parser.d.ts:322:5
    322     (options?: P & Partial<OptionFlag<T>>): OptionFlag<T | undefined>;
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
hm I see